src/l500/l535-preset-option.h

```diff
@@ -0,0 +1,23 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "hw-monitor.h"
+#include "l500-device.h"
+
+namespace librealsense
+{
+    namespace ivcam2
+    {
+        namespace l535
+        {
+            class l535_preset_option : public float_option_with_description< rs2_l500_visual_preset >
+            {
+            public:
```

Comment:
Use `super` because it repeats in the .cpp

Reply:
done